flake.nix

```diff
@@ -0,0 +1,77 @@
+{
+  description = "dynamic ceph provisioner";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+  };
+
+  outputs = inputs @ {
+    self,
+    flake-parts,
+    ...
+  }:
+
+  
+    flake-parts.lib.mkFlake {inherit inputs;} 
+
+    ({ withSystem, ... }: {
+      flake.overlays.default = final: prev:
```

Comment:
I'm not sure what's going on here. The overlay should look like `final: prev: final.buildGoModule [...]` (or something like that) and you should consume it in your package definition instead. Currently neither final nor prev appears anywhere on the RHS of =, so the package set used to compose overlays is never used.

Reply:
Thanks for helping me through this sticking point, should be good now :)